### PR TITLE
Remove unused codeact-SWE agent

### PR DESCRIPTION
**Pull request by enyst**
This PR removes the unused codeact-SWE agent and its references.

Changes:
- Removed the codeact-SWE agent directory
- Removed references in SWE-bench evaluation scripts
- Removed the SWE-specific prompt file

The codeact-SWE agent was a specialized version of CodeAct for solving GitHub issues, but it has not been maintained or updated in a while and is no longer used.

---

To run this PR locally, use the following command:
```
docker run -it --rm   -p 3000:3000   -v /var/run/docker.sock:/var/run/docker.sock   --add-host host.docker.internal:host-gateway   -e SANDBOX_RUNTIME_CONTAINER_IMAGE=docker.all-hands.dev/all-hands-ai/runtime:d1c7550-nikolaik   --name openhands-app-d1c7550   docker.all-hands.dev/all-hands-ai/openhands:d1c7550
```

### CODE REVIEW

File: evaluation/benchmarks/swe_bench/run_infer.py [hunk 1]
```diff
@@ -56,21 +55,9 @@ def _get_swebench_workspace_dir_name(instance: pd.Series) -> str:
...
-            instruction += (
-                f'--- BEGIN HINTS ---\n{instance.hints_text}\n--- END HINTS ---\n'
-            )
-        instruction += CODEACT_SWE_PROMPT.format(workspace_dir_name=workspace_dir_name)
-    else:
-        # Instruction based on Anthropic's official trajectory
-        # https://github.com/eschluntz/swe-bench-experiments/tree/main/evaluation/verified/20241022_tools_claude-3-5-sonnet-updated/trajs
+
+    # Instruction based on Anthropic's official trajectory
+    # https://github.com/eschluntz/swe-bench-experiments/tree/main/evaluation/verified/20241022_tools_claude-3-5-sonnet-updated/trajs
```
Comment (xingyaoww) [inline evaluation/benchmarks/swe_bench/run_infer.py:60]: Do we need to fix indentation here?

**Review by xingyaoww — APPROVED** [review]
Except for indentation, all LGTM

File: evaluation/benchmarks/swe_bench/run_infer.py (hunk 1 above)
Comment (enyst) [reply evaluation/benchmarks/swe_bench/run_infer.py:60]: I think it was in an else, and now it's not

**Comment by xingyaoww** [comment]
Please fix the indentation for `instruction = (...)` statement in `evaluation/benchmarks/swe_bench/run_infer.py`.

**Comment by openhands-agent** [comment]
[OpenHands](https://github.com/All-Hands-AI/OpenHands) started fixing the pr! You can monitor the progress [here](https://github.com/All-Hands-AI/OpenHands/actions/runs/12332537714).

File: evaluation/benchmarks/swe_bench/run_infer.py (hunk 1 above)
Comment (enyst) [reply evaluation/benchmarks/swe_bench/run_infer.py:60]: double checking

**Comment by openhands-agent** [comment]
Overview:
The indentation fix appears to be the only change made, which properly aligns the code with Python's standard formatting (4 spaces for main indentation, 8 for continuation lines). This change successfully resolves the previously reported indentation issue.

Status: ✓ RESOLVED
No remaining issues are apparent based on the provided information.

File: evaluation/benchmarks/swe_bench/run_infer.py (hunk 1 above)
Comment (openhands-agent) [reply evaluation/benchmarks/swe_bench/run_infer.py:60]: Openhands fix success summary


The AI has confirmed that the indentation issue has been fixed by properly indenting the `instruction = (...)` statement with 4 spaces and its continuation lines with 8 spaces, which follows Python's standard indentation style. This addresses the feedback about checking if the code was previously in an else block and ensuring proper indentation. The change maintains Python's code formatting standards while preserving the logical structure of the code.

**Comment by openhands-agent** [comment]
The workflow to fix this issue encountered an error. Please check the [workflow logs](https://github.com/All-Hands-AI/OpenHands/actions/runs/12332537714) for more information.

File: evaluation/benchmarks/swe_bench/run_infer.py (hunk 1 above)
Comment (enyst) [reply evaluation/benchmarks/swe_bench/run_infer.py:60]: So cool that it answered here!

**Pull request merged**